src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -1,107 +1,114 @@
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject.ChatPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject.ChatPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialVersionPage;
 
 public class ChatTestsStaff extends NewTestTemplate {
 
   private Credentials credentials = Configuration.getCredentials();
 
-  private ChatPageObject openChatForUser(String userName, String password) {
+  private ChatPage openChatForUser(String userName, String password) {
     WikiBasePageObject base = new WikiBasePageObject();
     base.loginAs(userName, password, wikiURL);
-    return base.openChat(wikiURL);
+    return new ChatPage().open();
   }
 
   @DontRun(env = {"sandbox"})
-  @Test(groups = {"ChatTestsStaff_001", "ChatStaff", "ChatTests"})
-  public void ChatTestsStaff_001_twoUserEnterChat() {
-    ChatPageObject chatUserOne =
+  @Test(groups = {"ChatStaff", "ChatTests"})
+  public void twoStaffUsersCanEnterChat() {
+    ChatPage chatUserOne =
         openChatForUser(credentials.userNameStaff, credentials.passwordStaff);
-    chatUserOne.verifyChatPage();
 
     switchToWindow(1);
     new SpecialVersionPage().open();
+    openChatForUser(credentials.userNameStaff2, credentials.passwordStaff2);
 
-    ChatPageObject chatUserTwo =
-        openChatForUser(credentials.userNameStaff2, credentials.passwordStaff2);
-    chatUserTwo.verifyChatPage();
     switchToWindow(0);
-
-    chatUserOne.verifyUserJoinToChatMessage(credentials.userNameStaff2);
+    Assertion.assertTrue(chatUserOne.isUserWelcomeMessageDisplayed(credentials.userNameStaff2), "WELCOME MESSAGE IS NOT DISPLAYED");
   }
 
   @DontRun(env = {"sandbox"})
-  @Test(groups = {"ChatTestsStaff_002", "ChatStaff", "ChatTests"})
-  public void ChatTestsStaff_002_verifySwitchingBetweenMainAndPrivateSections() {
-    ChatPageObject chatUserOne =
+  @Test(groups = {"ChatStaff", "ChatTests"})
+  public void verifyStaffUsersCanSwitchBetweenMainAndPrivateSections() {
+    ChatPage chatUserOne =
         openChatForUser(credentials.userNameStaff, credentials.passwordStaff);
 
     switchToWindow(1);
     new SpecialVersionPage().open();
-
-    ChatPageObject chatUserTwo =
+    ChatPage chatUserTwo =
         openChatForUser(credentials.userNameStaff2, credentials.passwordStaff2);
-
-    String userTwoMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
-    chatUserTwo.writeOnChat(userTwoMessage);
+    chatUserTwo.writeOnChat();
 
     switchToWindow(0);
-    chatUserOne.verifyMessageOnChat(userTwoMessage);
-
+    Assertion.assertTrue(chatUserOne.isMessageOnChat(), "MESAGE ON CHAT IS NOT DISPLAYED");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MESAGE ON CHAT IS NOT DISPLAYED" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)